setup.py

```diff
@@ -59,6 +59,7 @@ def run_tests(self):
     install_requires=[
         "wrapt",
         "msgpack-python",
+        "opentracing",
```

Comment:
we should not enforce this dependency because people can use our tracing system without OpenTracing (it's a plus). I'd suggest to define an extra dependency: http://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies

In this way:
* we don't force existing users to download and install another dependency
* users will keep track of the OpenTracing package versions, and we don't risk dependencies conflicts